### PR TITLE
New generic EasyConfig for Molpro-mpp-2015.1.3

### PR DESCRIPTION
**Pull request by valtandor**

### CODE REVIEW

File: easybuild/easyconfigs/m/Molpro/Molpro-mpp-2015.1.3.linux_x86_64_openmp.eb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+# no source URL available, requires registration to download
+sources = ['%(namelower)s-%(versionprefix)s%(version)s%(versionsuffix)s.sh']
+
+precompiled_binaries = True
+
+# license file - uncomment if a licence file is supplied by your site and
+# is valid for all users - the value of license_file may have to be changed
+#import os
+#license_file = os.path.join(os.getenv('HOME'), 'licenses', name, 'license.lic')
```
Comment (boegel) [inline easybuild/easyconfigs/m/Molpro/Molpro-mpp-2015.1.3.linux_x86_64_openmp.eb]: please change these two lines to (avoid doing `import os`)

```python
# license_file = HOME + '/licenses/%(name)s/license.lic'
```

**Comment by valtandor** [comment]
@boegel Done.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
gligar03.gligar.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/e6557b1ad7f8af03bc3a6a9b3a4a5dc9 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @valtandor!

**Pull request merged**